firstvoices/backend/tests/test_apis/base_media_test.py

```diff
@@ -687,6 +723,8 @@ def test_usages_field_base(self):
 
     @pytest.mark.django_db
     def test_usages_field_permissions(self):
+        if self.model == Document:
+            pytest.skip("Documents don't currently have usages.")
```

Comment:
No need to change this because I know it's very temporary, but for future reference I recommend skipping tests in the child test class instead, like this: https://github.com/First-Peoples-Cultural-Council/fv-be/blob/a905565821c2d121da6b17d2a1a628cb977610b5/firstvoices/backend/tests/test_apis/test_bulkvisibility_api.py#L75

It has better visibility (you can look at the document test class and see what it runs and what it skips), and it avoids having to check for interactions in all the other child classes.

Reply:
I was wondering what the best way to do this was. Thanks for the tip. Much appreciated!